yarn-error.log

```diff
@@ -0,0 +1,17518 @@
+Arguments: 
+  /usr/local/bin/node /usr/local/bin/yarn init
+
+PATH: 
+  /opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Library/Apple/usr/bin:/Library/Frameworks/Mono.framework/Versions/Current/Commands:/Users/nikita/Documents/pioneer/node_modules/.bin
+
+Yarn version: 
+  1.22.10
```

Comment:
Oh that shouldn't land in the PR ;)

Can you remove this file and add `*.log` to gitignore?

Reply:
Done